src/pyenphase/envoy.py

```diff
@@ -225,6 +226,7 @@ async def _request(
 
         url = self.auth.get_endpoint_url(endpoint)
         debugon = _LOGGER.isEnabledFor(logging.DEBUG)
+        request_start = time.time()
```

Comment:
Please avoid fetching time if debug is not on.  fetching time can be 10x more expensive on platforms that don't have VDSO support for fetching time.

Reply:
implemented